packages/client/hmi-client/src/workflow/ops/calibrate-julia/tera-calibrate-julia.vue

```diff
@@ -246,33 +475,35 @@ function addMapping() {
 
 	emit('update-state', state);
 }
+
 // Set up model config + dropdown names
-// Note: Same as calibrate-node
 watch(
 	() => modelConfigId.value,
 	async () => {
 		const { modelConfiguration, modelColumnNameOptions } = await setupModelInput(
 			modelConfigId.value
 		);
 		modelConfig.value = modelConfiguration;
-		modelColumnNames.value = modelColumnNameOptions;
+		modelColumnNames.value = modelColumnNameOptions?.map((state) => state.trim());
 	},
 	{ immediate: true }
 );
 
 // Set up csv + dropdown names
-// Note: Same as calibrate-node
 watch(
 	() => datasetId.value,
 	async () => {
 		const { filename, csv } = await setupDatasetInput(datasetId.value);
 		currentDatasetFileName.value = filename;
 		csvAsset.value = csv;
-		datasetColumnNames.value = csv?.headers;
+		datasetColumnNames.value = csv?.headers?.map((col) => col.trim());
```

Comment:
If we want to trim we will want to trim for ciemss too
Can change this in the setupDatasetInput

Reply:
i just updated the `headers` property directly so that we don't have to go touch the ciemss code